troposphere/__init__.py

```diff
@@ -318,153 +343,146 @@ class GenericHelperFn(AWSHelperFn):
     def __init__(self, data):
         self.data = self.getdata(data)
 
-    def JSONrepr(self):
-        return self.data
+    def to_dict(self):
```

Comment:
Given the code duplication I had been contemplating moving the generic JSONrepr() into the AWSHelperFn base class. I think the same could be done for to_dict().